net-misc/freerdp/freerdp-2.11.1.ebuild

```
@@ -52,7 +52,9 @@ RDEPEND="
 	gstreamer? (
 		media-libs/gstreamer:1.0
 		media-libs/gst-plugins-base:1.0
-		x11-libs/libXrandr
+		X? (
+			x11-libs/libXrandr
+		)
```

Please collapse this to a single line.
```suggestion
		X? ( x11-libs/libXrandr )
```